index.html

```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
     <div class="main-container">
-        <img src="images/image-qr-code.png" alt=""> <br>
+        <img src="images/image-qr-code.png" alt="">
         <h3>Improve front-end skills by building projects</h3>
         <p>Scan the qr code to visit Frontend Mentor and take coding skills to the next level</p>
     </div>
```

Comment:
🌸 Mainly, I would like to congratulate you on writing clean HTML code. you did a good job!